File: 2058-concatenation-of-array/concatenation-of-array.py
class Solution(object):
    def getConcatenation(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """

        x = []

        for i in nums:
            x.append(i)
            
            
        for j in nums:
            x.append(j)
            

        return x
        

File: 2058-concatenation-of-array/concatenation-of-array.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        
        vector<int> x;

        for (int i = 0; i < nums.size(); i = i + 1) 
        {
            x.push_back(nums[i]);
        }

        for (int i = 0; i < nums.size(); i = i + 1) 
        {
            x.push_back(nums[i]);
        }

        return x;
    }
};